app/views/shared/google_analytics/_page_tracking.html.erb

```diff
@@ -0,0 +1,15 @@
+<% nonce = content_security_policy_script_nonce %>
+
+<%= nonced_javascript_tag do %>
+  var nonce = "<%= nonce %>";
+
+  (function(i,s,o,g,r,a,m,n){n=a;i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.setAttribute('nonce',n);a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga', nonce);
+
+  ga('create', '#{Figaro.env.google_analytics_key}', 'auto');
```

Comment:
This should be `'<%= Figaro.env.google_analytics_key %>'` since you converted it from Slim to ERB.